contrib/dockerd-rootless-setuptool.sh

```diff
@@ -385,6 +390,9 @@ cmd_entrypoint_install() {
 		cli_ctx_create "${CLI_CONTEXT}" "unix://${XDG_RUNTIME_DIR}/docker.sock" "Rootless mode"
 	fi
 
+	INFO "Use CLI context \"${CLI_CONTEXT}\""
+	cli_ctx_use "${CLI_CONTEXT}"
+
```

Comment:
> L404: echo "export DOCKER_HOST=unix://${XDG_RUNTIME_DIR}/docker.sock"

This line is no longer necessary. Maybe prepend `echo `Some applications may require the following environment variable too:`.

Reply:
i will. I'm curious, when should it be necessary?

Reply:
Third party applications that do not support the CLI context.

e.g., https://github.com/docker-java/docker-java